scheds/rust/scx_tickless/src/bpf/main.bpf.c

```diff
@@ -371,46 +402,36 @@ static int sched_timerfn(void *map, int *key, struct bpf_timer *timer)
 	}
 
 	bpf_timer_start(timer, tick_interval_ns(), 0);
-	bpf_rcu_read_unlock();
 
 	return 0;
 }
 
 /*
- * Start the BPF timer on a target CPU (if not already started).
+ * Start the BPF timer on a target CPU.
  */
-static void fire_timer(s32 cpu)
+static void init_timer(s32 cpu)
 {
 	struct cpu_ctx *cctx;
 	int ret;
 
+	if (!is_primary_cpu(cpu))
```

Comment:
I had to look through the timer docs again, it's really annoying that `bpf_timer_start` only can pin with `BPF_F_TIMER_CPU_PIN` and that only works with the caller CPU.

Reply:
Yeah, apparently there's not a way to pin the timer to CPU N, where N != current CPU.